### PR TITLE
fix shift duration timestamp in join game menu and check antags menu

### PR DESCRIPTION
**Pull request by nervere**
## What this does
title

## Why it's good
currently it displays the time since init finished, which is incorrect
[bugfix]
## Changelog

:cl:
 * bugfix: The "Join Game" and "Check Antagonist" menus now display the proper shift duration.

### CODE REVIEW

File: code/modules/mob/new_player/new_player.dm [hunk 1]
```diff
@@ -525,7 +521,7 @@
 		.manifest tr.requested_department td {background-color: #00FF00}
 		.manifest th.reqhead td {background-color: #844}
 		.manifest tr.reqalt td {background-color: #FCC}
-		</style></head><body><center>Round Duration: [round(hours)]h [round(mins)]m<br>"}
```
Comment (boy2mantwicethefam) [inline code/modules/mob/new_player/new_player.dm:528]: Undocumented text change from round to shift

Comment (nervere) [reply code/modules/mob/new_player/new_player.dm:528]: Not nearly significant enough for a changelog entry.

**Review by west3436 — APPROVED** [review]

**Comment by jwhitak** [comment]
This bugfix changes server-standard verbiage incorrectly - our codebase uses Round as the correct noun to describe the game's duration. This is used in multiple places throughout the code, notably in Dynamic as a description for when antagonists fire. Shift is not used in the code at any point, though many players use the term to talk about the concept in an in character sense.

That said, I personally don't actually care enough to not merge a useful bugfix. If someone else does care they can make a PR to change it.

**Comment by SonixApache** [comment]
#36488

**Pull request merged**